core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -86,16 +85,15 @@ protected CachingCatalog(Catalog catalog, boolean caseSensitive, long expiration
   }
 
   /**
-   * CacheWriter class for removing metadata tables when their associated data table is expired
+   * RemovalListener class for removing metadata tables when their associated data table is expired
    * via cache expiration.
+   *
+   * @see com.github.benmanes.caffeine.cache.RemovalListener
```

Comment:
Nit: This `@see` doesn't seem necessary  to me given that we're directly implementing `RemovalListener` on the next line.

Reply:
removed